modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/InstanceSettingsControlPanelEntry.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.web.internal.portlet;
+
+import com.liferay.configuration.admin.constants.ConfigurationAdminPortletKeys;
+import com.liferay.portal.kernel.portlet.ControlPanelEntry;
+import com.liferay.portal.kernel.portlet.OmniadminControlPanelEntry;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Christopher Kian
+ */
+@Component(
+	immediate = true,
+	property = "javax.portlet.name=" + ConfigurationAdminPortletKeys.INSTANCE_SETTINGS,
+	service = ControlPanelEntry.class
+)
+public class InstanceSettingsControlPanelEntry
+	extends OmniadminControlPanelEntry {
```

Comment:
Hi @ChrisKian, I think this should extend the `AdministratorControlPanelEntry` since the portlet is checking for company admin permission. 

https://github.com/liferay/liferay-portal/blob/master/modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/InstanceSettingsPortlet.java#L92-L98

Reply:
Thanks for finding that @pei-jung.  I have force-pushed the change